use_cases/gretel.json

```diff
@@ -290,6 +309,20 @@
         "trainingTime": "< 1 min",
         "bytes": 65300
       }
+    },
+    {
+      "gtmId": "use-case-image-synthetics",
```

Comment:
**quick question (nonblocking):** i noticed this one is now below transform v1, which we previously pushed to the bottom, so just wanted to make sure the ordering was intentional :)

Reply:
Yes, we're not supporting the image synthetics app right now; the model's out of date. We're close to deprecating transform v1 but it does have utility for NER, so I figure we can keep it above image synthetics. It's not a huge deal though, it doesn't look like many users click through to the blueprints page.